fdbrpc/tests/CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
-if(NOT WIN32)
+if (NOT WIN32)
```

Comment:
If, else, endif etc are all function calls. So the whitespace shouldn’t be there